README.md

```diff
@@ -1 +1,81 @@
 # Template Marlin Processor
+
+A template Marlin package with a skeleton processor. It contains the following:
+
+* A template structure for organizing your code.
+* A CMakeLists.txt for compilation.
+* A skeleton processor that you can modify.
+
+Note that official Marlin packages follow a slightly different template. This is because they are based on old CMake conventions. The template documented here has been updated for modern CMake.
+
+## Compiling
+Start by making a fork or directly cloning the template repository.
+
+```bash
+cd /scratch/${USER}/tutorial2024
+git clone https://github.com/MuonColliderSoft/TemplatePackage.git
+```
+
+The package can be compiled by loading the Muon Collider software environment and compiling as any CMake package.
+
+```
+# cmake -S TemplatePackage -B build
+cmake -S TemplatePackage -B build -DILC_HOME=
```

Comment:
Is `-DILC_HOME=` required? When I was trying to compile it earlier, I didn't use it and didn't have any errors.

Reply:
yes, with the last update this "workaround" is not needed. Removed